### PR TITLE
Status checker

### PR DESCRIPTION
**Pull request by sudach**
## What?
- Commands for checking environment variables and DB access(only read check).
`pydtk status env`, `pydtk status access`

## Why?
- Make it easier to check the above.

## See also [Optional]
- https://github.com/dataware-tools/pydtk/issues/71

/kind feature
/assign @sudach

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/85#" title="Author self-approved">sudach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [sudach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by d-hayashi — APPROVED** [review]
実装あざます！
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/85#pullrequestreview-720976948" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/85#" title="Author self-approved">sudach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,sudach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pydtk/bin/cli.py
Comment (d-hayashi) [inline pydtk/bin/cli.py:39]: すいません、ここだけ修正お願いします

**Comment by hdl-service** [comment]
New changes are detected. LGTM label has been removed.

**Pull request merged**
